### ISSUE TITLE
Optimize the acquisition funnel for the IDE extension

### ISSUE THREAD

**Issue by rrhyne** [sourcegraph/sourcegraph#27948]
The IDE extension is critical for helping developers surface code where they work most often. Currently, most visitors are unaware of this tool. 

Optimize pre-auth and post auth getting started flows to ensure all users are aware of the existence of the extension and are routed to the landing page.  

UPDATE:

Figma file containing creative, copy and metrics is available here. 

https://www.figma.com/file/VlpvIoBXrG6GFWsEFmc6rB/Browser-extension-CTA-improvements?node-id=71%3A8094

**Comment by vdavid** [comment]
@rrhyne, this would make sense as a follow-up project to #27100. But I'll need some specifics and design work first. Can you provide these?

**Comment by rrhyne** [comment]
Yes, what kind of time frame do you anticipate needing for the assets?

**Comment by vdavid** [comment]
@rrhyne **Early next week.**
I think we'll be done with "Improve browser extension acquisition" this week, so I could start working on this next week.

**Comment by vdavid** [comment]
Hey @rrhyne!
I might be done with my current issue today, and this one would be next up.
I've invited you for a short meeting about this one today to make sure we make progress.
We can either do the meeting and figure it out there or feel free to provide a design and your thoughts here async.
I'm okay with either.

**Comment by vdavid** [comment]
I talked with Rob. Here is an update:
- This is a very similar issue to #27100. We want to advertise our IDE extension to both visitors **and** signed-in users.
- We'll **alternate them** with the other CTAs: all will appear on every 6th page view, and these new ones will be shifted by three page views. (We identified multiple ways to make it work with the rest of the CTAs. Rob made designs to fit them on top of each other and next to each other. It was a bit too much, though, so we decided to alternate them.)
- Design and copy: We've updated the [Figma file](https://www.figma.com/file/VlpvIoBXrG6GFWsEFmc6rB/Browser-extension-CTA-improvements) to include this component.
- As a next step, development can start, plus @rrhyne will add the final copy, metric names, and UTM parameters.

**Comment by rrhyne** [comment]
Copy, illustrations and metrics definitions are here:

https://www.figma.com/file/VlpvIoBXrG6GFWsEFmc6rB/Browser-extension-CTA-improvements?node-id=71%3A8094

**Comment by vdavid** [comment]
@rrhyne What should be the URL for the main CTA button ("Learn more about IDE extensions")?

Also, an idea: as @erzhtor pointed out in today's meeting, we won't know whether the user has **already installed** an IDE extension. I suppose once a user has it installed, they'll close the alert. To get more info, upon clicking the "Close" button, we could show **a small menu** with ~three items like "I'm not interested" / "I already have it installed" / "Just let me close this". This is probably something for later (if it's even a good idea), just wanted to share it.

**Comment by vdavid** [comment]
Pings are still missing.
